Keas.Jobs.Core/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Keas.Jobs.Core
+{
+    public class Class1
+    {
+    }
+}

```

Comment:
Is this class needed? I don't see it used, and the name is suspicious or not actually being used..

Reply:
@jcubbage No, but if we add Serilog, it will be replaced with a log helper class. This was just the default that got created when I created the job. I'll delete it.